resources/benchmark-runner.mjs

```diff
@@ -33,13 +33,10 @@ class Page {
         return new Promise((resolve) => {
             const resolveIfReady = () => {
                 const element = this.querySelector(selector);
-                if (element) {
-                    window.requestAnimationFrame(() => {
-                        return resolve(element);
-                    });
-                } else {
-                    setTimeout(resolveIfReady, 50);
-                }
+                let callback = resolveIfReady;
```

Comment:
This is a nit, but I'd find it simpler to follow if kept the basic structure in place and just replaced the setTimeout with a rAF

```js
const resolveIfReady = () => {
  const element = this.querySelector(selector);
  if (element) {
    window.requestAnimationFrame(() => {
      resolve(element);
    });
  } else {
    window.requestAnimationFrame(resolveIfReady);
  }
};
```

Alternatively we could refactor this to only look for the element inside the rAF, which could potentially lead to one less rAF if the element does end up existing after the first rAF is done.

```js
const resolveIfReady = () => {
  window.requestAnimationFrame(() => {
    const element = this.querySelector(selector);
    if (element) {
      resolve(element);
    } else {
      resolveIfReady();
    }
  });
};
```

just

Reply:
There is no real benefit to delay the lookup at this point, since the only workload where the content isn't there yet is the perf dashboard.
Eventually I'd like to remove the additional rAF delay here alltogether if it's not needed (but I prefer doing this separate to be able to easily measure the perf impact). WDYT?